pagopa-ecommerce-transactions-service/transactions-service.env

```diff
@@ -36,11 +36,10 @@ API_CONFIG_CONNECTION_TIMEOUT=10000
 API_CONFIG_PSP_UPDATE_CRON_STRING="*/60 * * * * *"
 
 ECOMMERCE_STORAGE_CONNECTION_STRING="DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;QueueEndpoint=http://storage:10001/devstoreaccount1"
-TRANSACTION_AUTHORIZATION_REQUESTED_EVENT_QUEUE_NAME=transactionauthrequestedtevents
-TRANSACTION_AUTHORIZATION_REQUESTED_EVENT_QUEUE_VISIBILITY_TIMEOUT=60
-TRANSACTION_ACTIVATED_EVENT_QUEUE_NAME=pagopa-ecommerce-transactions-expiration-queue
-TRANSACTION_CLOSURE_SENT_EVENT_QUEUE_NAME=pagopa-ecommerce-transactions-close-payment-retry-queue
-TRANSACTION_REFUND_QUEUE_NAME=pagopa-ecommerce-refund-queue
+
+TRANSACTION_EXPIRATION_QUEUE_NAME=expiration-queue
+TRANSACTION_CLOSEPAYMENT_RETRY_QUEUE_NAME=closepayment-retry-queue
+TRANSACTION_REFUND_QUEUE_NAME=refund-queue
```

Comment:
Align this queue names with the one configured for dispatcher

Reply:
Fixed in d984a2d